src/frontend/tslint.json

```diff
@@ -5,6 +5,7 @@
   "rules": {
     "arrow-parens": false,
     "interface-name": [true, "never-prefix"],
+    "max-line-length": false,
```

Comment:
Are you sure you really want to disable it ? Why did you choose to do that ? Why not allowing a higher length ?

Reply:
As we use `prettier`, having this rule in `tslint` is 99% pointless: `prettier` will break down long lines for us (and the prettier check will ensure we ran it).

As for the 1%: there is only one case where `prettier` does not break down a long line (a very long import with a single imported item). As a general policy I think it's best to prioritize `prettier` over `tslint`, and so that's what we're doing by disabling this check.